src/flamenco/features/feature_map.json

```diff
@@ -168,7 +168,7 @@
   {"name":"require_rent_exempt_split_destination","pubkey": "D2aip4BBr8NPWtU9vLrwrBvbuaQ8w1zV38zFLxx4pfBV"},
   {"name":"better_error_codes_for_tx_lamport_check","pubkey": "Ffswd3egL3tccB6Rv3XY6oqfdzn913vUcjCSnpvCKpfx"},
   {"name":"enable_alt_bn128_compression_syscall","pubkey": "EJJewYSddEEtSZHiqugnvhQHiWyZKjkFDQASd7oKSagn"},
-  {"name":"programify_feature_gate_program","pubkey": "8GdovDzVwWU5edz2G697bbB7GZjrUc6aQZLWyNNAtHdg"},
+  {"name":"programify_feature_gate_program","pubkey": "8GdovDzVwWU5edz2G697bbB7GZjrUc6aQZLWyNNAtHdg","hardcoded":1},
```

Comment:
I think this has been "reverted" (i.e. never activated) rather than "cleaned up" (on by default). We probably need a way of distinguishing this in our map, otherwise things will get confusing.

Reply:
fixed